src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RSA/ImportExport.cs

```diff
@@ -248,7 +248,10 @@ public static void ImportNoExponent()
 
             using (RSA rsa = RSAFactory.Create())
             {
-                Assert.ThrowsAny<CryptographicException>(() => rsa.ImportParameters(imported));
+                if (rsa is RSACryptoServiceProvider)
+                    Assert.ThrowsAny<CryptographicException>(() => rsa.ImportParameters(imported));
+                else
+                    Assert.ThrowsAny<ArgumentException>(() => rsa.ImportParameters(imported));
```

Comment:
Summarizing a thread where we tried to figure out why netfx' RSACng throws a different exception than RSACsp:  Let's declare CryptographicException the right answer.  If there are tests we have to disable for netfx because of that, let's do so.  And please make sure we get a bug in netfx to evaluate changing RSACng behavior to match RSACsp for these cases.  (Just open an issue with a summary, mark it netfx-port-consider, then hit close).